test/CollapseSpec.js

```diff
@@ -40,13 +40,16 @@ describe('<Collapse>', () => {
   describe('from collapsed to expanded', () => {
     beforeEach(() => {
       wrapper = mount(<Component>Panel content</Component>);
-
       // since scrollHeight is gonna be 0 detached from the DOM
-      sinon
-        .stub(wrapper.instance().collapse, '_getScrollDimensionValue')
+      getScrollDimensionValue = sinon
+        .stub(collapseHelpers, '_getScrollDimensionValue')
         .returns('15px');
```

Comment:
looking at this again – can we just stub out `scrollX` or `scrollY` or whatever on the base node class?

Reply:
I'm not sure to understand... I tried to stub the scrollHeight, but I didn't found a way to achieve it 😢